### PR TITLE
Update CombC.schelp

### PR DESCRIPTION
**Pull request by tiagmoraismorgado**
a huge thanks to @snappizz

### CODE REVIEW

File: HelpSource/Classes/CombC.schelp [hunk 1]
```diff
@@ -62,14 +62,7 @@ code::
...
 // large delay lines
-(
-s.options.memSize= 8192*2;
-s.reboot;
-)
-(
-s.options.memSize= 8192*20;
-s.reboot;
-)
+(s.options.memSize= 8192*20; s.reboot;)
```
Comment (telephon) [inline HelpSource/Classes/CombC.schelp]: for a single line, you need no parentheses.

**Comment by tiagmoraismorgado** [comment]
maybe we can tweak it a little bit. I already started doing so. I am going to make a new commit. you're welcome to come onboard on the reviewing process

**Comment by tiagmoraismorgado** [comment]
some of the issues you mention have already been fixed

**Comment by tiagmoraismorgado** [comment]
so, how would you suggest us to fix that?

**Comment by tiagmoraismorgado** [comment]
anyway, feel free to delete this pull request if you don't feel like this is a proper PR

**Comment by redFrik** [comment]
well, i think the example that was there before that you deleted was pretty good.  also remember that you committed it yourself (after some discussions) in June 2016.  why do you want to delete it and change it to something worse (in my opinion).  can you describe in what way this new example is better showing how the CombC ugen works?

**Comment by tiagmoraismorgado** [comment]
the code somewhat does something pretty similar, but I think that the major update was in the supercollider syntax, and the timings in the arrays. if you feel however that it doesn't make sense to use this tweak we can discard ourselves of this PR

**Pull request closed without merging**